persistence/src/main/java/com/kodemon/persistence/entity/Trainer.java

```diff
@@ -40,6 +42,16 @@
     @NotNull
     private String lastName;
 
+    @Column(nullable = false)
+    @NotNull
+    @ColumnDefault("false")
```

Comment:
why this annotation? Something like 
`private boolean isAdmin = false;` didn't work?
This is not very good, as the annotation is hibernate-specific, so it creates tight coupling with the framework.
...
Same with `isBlocked`